### ISSUE TITLE
scroll not down 

### ISSUE THREAD

**Issue by PAssssAT** [xbsoftware/enjoyhint#114]

**Comment by ShvedkaSvetka** [comment]
Hi @PAssssAT,
We need more information to reproduce your issue. Could you pls provide information about the device/ browser and etc. Thank you.

**Comment by BizzottoClaudio** [comment]
Instead of opening a new one i'll use this one.
I'm on chrome 78.0.3904.87 (official build) (64 bit)
As can be seen from the gif, at some point the tutorial moves at the bottom but the browser doesn't react.
Also you can see that i press the (last visible) button twice because at the first click the button highlight with no actions.

![MUrS9mp6ak](https://user-images.githubusercontent.com/20600987/68412384-68970b80-018c-11ea-8553-6751954f92c0.gif)

**Comment by GIOkafor** [comment]
@BizzottoClaudio were you able to figure it out?

Same thing happens in my project, the scroll seems to work fine on desktop but on mobile it shows the next text but doesn't move the page.

**Comment by BizzottoClaudio** [comment]
@GIOkafor nope, i haven't used the library in the last years, but i find it a good choice nonetheless :)

**Comment by tsemashkoXB** [comment]
Hi @PAssssAT,

thanks for reporting this, and sorry for the long silence.

Scrolling to off-screen step targets was fixed — please upgrade when you can and let us know if anything still looks off.
